src/Pages/Sign_in_up/signin.js

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import { withRouter } from "react-router-dom";
+import "./signin.scss";
+
+class Signin extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      user_name: "",
+      password: "",
+    };
+  }
+
+  handleInput = (e) => {
+    // console.log("onChange");
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  goToMain = () => {
+    this.props.history.push("/Main");
+  };
+
+  clickSignin = () => {
+    fetch("http://10.58.1.192:8000/user/signin", {
+      method: "POST",
+      body: JSON.stringify({
+        user_name: this.state.user_name,
+        password: this.state.password,
+      }),
+    })
+      .then((res) => res.json())
+      // .then((result) => console.log("결과: ", result));
+      .then((res) => this.testSignin(res));
+  };
+
+  testSignin = (res) => {
+    //console.log(res)
+    if (res.message === "SIGNIN_SUCCESS") {
+      alert("로그인 성공");
+    } else {
+      alert("로그인 실패");
+    }
```

Comment:
단일 if, else문이네요! 
아래와 같이 삼항연산자를 활용해보면 어떨까요?
```suggestion
    alert(res.message === "SIGNIN_SUCCESS" ? "로그인 성공" : "로그인 실패");
    }
```

Reply:
적용해보겠습니다